### DISCUSSION TITLE
antv s2单元格内部是否可以支持写入html？

### DISCUSSION THREAD

**Discussion by dabaoRain** [antvis/S2#2714] [Q&A]
antv s2 v1版本透视表 tree结构，一个单元格内的数据是 200（87.26%），我想设置200 和 87.26的文本颜色不一致，目前能想到的是单元格内可以插入div加入样式，写入html的方法试了很多没有成功的，不知道有没有其他方法可以实现？

**Comment by lijinke666** [comment]
不可以.

你可以自定义单元格, 先把 "200（87.26%）" split 成 ['200', '(87.26%)'], 先绘制 200, 然后获取到文本坐标, 在 `200` 这个文本 (textshape) 后面再绘制一个文本 (textshape)

https://github.com/antvis/S2/blob/1ac36e8079397db450ef7ab87ac2cd09ee973377/packages/s2-core/src/cell/base-cell.ts#L272-L279

这样才能控制不同的颜色, 请自行查阅相关文档

**Reply by dabaoRain** [reply]
嗯嗯，还是得自定义单元格内容，谢谢大佬，
